src/main/java/org/parabot/Landing.java

```diff
@@ -135,6 +135,12 @@ private static void parseArgs(String... args) {
                 case "-no_validation":
                     Core.disableValidation();
                     break;
+                case "-nobdn":
+                    Core.setLoadBdn(false);
+                    break;
+                case "-skiplogin":
```

Comment:
Isn't this exactly what `-debug` does?

Reply:
-debug results in an NPE beacuse it tries to load BDN servers, but debug skips logging in all together. This solves but also adds more flexibility.

Reply:
Shouldn't we fix the debug error instead of working around it then?

Reply:
This PR does fix it, and also adds -skiplogin which debug previously acted as